blog/_posts/2018-03-19-accessible-scala.md

```diff
@@ -55,11 +55,16 @@ type a context bar
 from s string to int
 ```
 
-During the Scala Center Advisory Board last week, a proposal authored and
-strongly advocated by Sam Halliday, with help from Rui Batista, was accepted as
-a recommendation by the board, with a unanimously positive vote from all the
-members, including 47 Degrees, IBM, Goldman Sachs, Lightbend, Morgan Stanley,
-SAP, Twitter and Verizon.
+During the [Scala Center](https://scala.epfl.ch/) Advisory Board last week, a
+proposal authored and strongly advocated by [Sam
+Halliday](https://github.com/fommil), with help from [Rui
```

Comment:
I'd prefer my gitlab account if that's ok. It's a more ethical software repository meeting the FSF guidelines.